Don't install .pc file: it would conflict between installs for
multiple python versions. In case it is needed, something must be found to
handle name conflicts.

--- bindings/python/CMakeLists.txt.orig	2024-05-27 21:47:00.000000000 +0200
+++ bindings/python/CMakeLists.txt	2024-05-28 10:16:31.992490625 +0200
@@ -270,14 +270,6 @@
     set(_PKG_CONFIG_PYWRAP_CFLAGS "${_PKG_CONFIG_PYWRAP_CFLAGS} ${cflags}")
   endforeach()
 
-  configure_file("${CMAKE_CURRENT_SOURCE_DIR}/pinocchiopy.pc.cmake"
-                 "${CMAKE_CURRENT_BINARY_DIR}/pinocchiopy.pc")
-
-  install(
-    FILES "${CMAKE_CURRENT_BINARY_DIR}/pinocchiopy.pc"
-    DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig
-    PERMISSIONS OWNER_READ GROUP_READ WORLD_READ OWNER_WRITE)
-
   if(DOXYGEN_FOUND AND DOXYGEN_VERSION VERSION_GREATER 1.8.17)
     set(DOXYGEN_GENERATE_HTML YES)
     set(DOXYGEN_GENERATE_LATEX NO)
